cookbooks/cdo-freegeoip/.kitchen.yml

```diff
@@ -0,0 +1,16 @@
+---
+transport:
+  name: sftp
+driver:
```

Comment:
Is this consistent with the known compatibility limitation of not running inside a docker container?
Note that I don't know what I'm talking about here.